plugins/outputs/http/README.md

```diff
@@ -104,6 +107,15 @@ format by default.
   # non_retryable_statuscodes = [409, 413]
 ```
 
+### Google API Auth
+
+Configuring the `credentials_file` to the path to the json key allows Telegraf's http output plugin to communicate with
+Google Cloud APIs. To learn about creating Google service accounts, consult Google's
+[oauth2 service account documentation][create_service_account]. An example use case is a metrics proxy deployed to
+Cloud Run. In this example, the service account must have the "run.routes.invoke" permission.
```

Comment:
```suggestion
The `credentials_file` setting is used with Google Cloud APIs. It specifies the json key file. To learn about creating Google service accounts, consult Google's
[oauth2 service account documentation][create_service_account]. An example use case is a metrics proxy deployed to
Cloud Run. In this example, the service account must have the "run.routes.invoke" permission.
```

Reply:
@zachmares Did we address this?

Reply:
Committed the suggested changes. Thank you @reimda for the word-smithing!